cmd/loop/instantout.go

```diff
@@ -0,0 +1,165 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/lightninglabs/loop/instantout/reservation"
+	"github.com/lightninglabs/loop/looprpc"
+	"github.com/urfave/cli"
+)
+
+var instantOutCommand = cli.Command{
```

Comment:
Would be cool to have a cli command to also list available reservations. (can be a follow up).